### PR TITLE
write_touchstone: write port names to file

### PR DESCRIPTION
**Pull request by luar123**
If port names are available in Network.port_names, write these to the touchstone file as '! Port[#] = name'

### CODE REVIEW

File: skrf/network.py [hunk 1]
```diff
@@ -1955,6 +1955,16 @@ def get_buffer():
...
             output.write('# {} S {} R {} \n'.format(self.frequency.unit, form, str(abs(self.z0[0, 0]))))
 
+            # write ports
+            try:
+                if self.port_names and len(self.port_names) == self.number_of_ports:
+                    ports = ''
+                    for port_idx, port_name in enumerate(self.port_names):
+                        ports += '! Port[{}] = {}\n'.format(port_idx, port_name)
+                    output.write(ports)
+            except(AttributeError):
```
Comment (jhillairet) [inline skrf/network.py:1965]: I would suggest using the more pythonic :
`except AttributeError:`

**Comment by jhillairet** [comment]
Thank you for your contribution. Do you think you could add a specific test for this improvement?

File: skrf/network.py (hunk 1 above)
Comment (luar123) [reply skrf/network.py]: Thanks for the suggestion, I have changed it throughout the write_touchstone() method. And also fixed some white space formating warnings.

Comment (jhillairet) [reply skrf/network.py]: great, thank you.

**Comment by jhillairet** [comment]
thank you for the improvement

**Pull request merged**